.github/workflows/firmware-hil-nightly.yml

```
@@ -0,0 +1,79 @@
+    # Nightly at 03:00 UTC -- avoids peak GitHub Actions queue times
+    - cron: '0 3 * * *'
+  push:
+    # Also run on release tags so every shipped firmware has HIL sign-off
+    tags: ['v*']
+  workflow_dispatch:     # manual trigger for on-demand HIL runs
+
+jobs:
+  hil-test:
+    name: HIL Integration Tests on RX72N
+    # DISABLED: remove this line when runner is registered
+    if: false
```

actionlint (1.7.11)

[error] constant expression "false" in condition. remove the if: section

(if-cond)